Draft.md

```diff
@@ -75,31 +109,64 @@ _draft_
 - The **Secretary** will keep a record of meetings and member contact info, and will create a summary of the meeting and upload that summary to Github after the meeting is concluded. They will send out an invite to club members before the club meets
 - The **Discord Admins** will organize and create rules for the Discord, approve changes to the Discord bot, and be vested with the power to kick individuals from the discord for legitimate reasons.
 
+**_Option_**:
+
+- The **Treasurer** will track expenses, investigate fundraising activities, submit requests for funds, and allocate funds to projects.
+
+**_Option_**:
+
+- The **Quartermaster** shall budget and plan events, ensure enough supplies are acquired, and organize and direct the trip safely.
+
+
 **Section X - Nomination**:
 
+**_Option 1_**: 
+
+1. Members may nominate themselves as candidates any time before an election.
+  
+**_Option 2_**:
+
+1. Another member must nominate a candidate prior to an election for them to be eligible.
```

Comment:
i like this idea, it may however be harder to pull off in small groups. perhaps set this as default intill a member number threshold is reached.